nipype/interfaces/afni/preprocess.py

```diff
@@ -1348,7 +1349,8 @@ class ECMInputSpec(CentralityInputSpec):
     )
     eps = traits.Float(
         desc="sets the stopping criterion for the power iteration; "
-        "l2|v_old - v_new| < eps*|v_old|; default = 0.001",
+        ":math:`l2\\|v_\\text{old} - v_\\text{new}\\| < eps\\|v_\\text{old}\\|`; "
```

Comment:
I think this will make `ECM.help()` look worse.

```
        eps: (a float)
                sets the stopping criterion for the power iteration;
                :math:`l2\|v_\text{old} - v_\text{new}\| < eps\|v_\text{old}\|`;
                default = 0.001
                argument: ``-eps %f``
```

Reply:
Ugh yes. I'll try to think how we can have both. For now, I believe the help is consumed via documentation more often. Thoughts:

* What if `help()` was limited to one line?
* Alternatively, what about having a new `help` metadata for building help that overrides `desc` when present?

Reply:
@satra, wdyt about adding more metadata entries?

Reply:
I would probably prefer to detect `:math:` and do something intelligent with it, myself. In the short term, I think reverting to the text version in the AFNI docs is probably cleanest to ensure readability. For improved readability, we could either escape them or convert to Unicode where helpful.

Maybe let's move converting to `:math:` and handling it well in `help()` to a separate issue, just to keep this one moving.

Reply:
#3126